packages/network-controller/src/rpc-service/rpc-service.ts

```diff
@@ -144,7 +197,7 @@ export class RpcService implements AbstractRpcService {
   /**
    * The URL of the RPC endpoint.
    */
-  readonly #endpointUrl: URL;
+  readonly endpointUrl: URL;
```

Comment:
This is a new change. I made this public because when we cut over to a failover RPC service, we need to pass the endpoint URL of the failover to `onBreak` event listeners.